cmd/main.go

```diff
@@ -55,7 +63,7 @@ var cmd = &cobra.Command{
 		for _, envVar := range opts.envVariables {
 			splitVar := strings.SplitN(envVar, "=", 2)
 			if len(splitVar) != 2 {
-				cmd.Usage() //nolint
+				cmd.Usage() // nolint
```

Comment:
The lack of space is intentional https://github.com/golangci/golangci-lint/issues/3109#issuecomment-1218842579

Reply:
Although lint seems to pass with the space, not sure why, but directive format in go is // followed by the directive directly without whitespace.

Reply:
Yeah, the change wasn't intentional on my part.  But goland is pretty intentional about it...

Reply:
Weird, golang doesn't touch it for me.

Reply:
Are you using a go locally less than 1.19?

Reply:
go version go1.20 linux/amd64

Reply:
I can turn Reformat on save off

Reply:
I swear lint was failing with the space when I changed it, but if it's passing now we can fix it again if it becomes a problem